option.go

```diff
@@ -51,14 +54,24 @@ func WithPreviewWindow(f func(i, width, height int) string) Option {
 	}
 }
 
-// WithHotReload reloads the passed slice automatically when some entries are appended.
+// Deprecated: WithHotReload reloads the passed slice automatically when some entries are appended.
 // The caller must pass a pointer of the slice instead of the slice itself.
```

Comment:
[nits] Let the caller know the alternative option!

```suggestion
// WithHotReload reloads the passed slice automatically when some entries are appended.
// The caller must pass a pointer of the slice instead of the slice itself.
//
// Deprecated: use WithHotReloadLock instead.
```

Reply:
Make sense! I am too used to auto-completion :)